docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV PYTHONUNBUFFERED=1 \
 # hadolint ignore=DL3005,DL3008,DL3013
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install --no-install-recommends -y git mime-support curl && \
+    apt-get install --no-install-recommends -y git mime-support curl libxml2 && \
```

Comment:
Do we need to similarly add any other packages to support the optional features "out of the box"? Comparing against line 33 below, perhaps `openssl`, `default-libmysqlclient`, `libldap`, `libsasl2`?

Reply:
@glennmatthews This fixes the initial issue of nautobot not even starting with the default config.  You bring up a good point for the other dependencies `openssl`, `openldap`, and `libsasl2` are included in the default image so no need to install those for the final image, although I don't have an ldap environment to test.  It does appear MySQL is going to be an issue, I'm looking for the right package now, of course it can't follow the same naming convention of the other packages...

Reply:
turns out we need the same mysql lib in prod as development images so I moved it to base.